meta-genivi-dev/recipes-dev-platform/packagegroups/packagegroup-gdp-hmi.bb

```diff
@@ -22,4 +22,5 @@ RDEPENDS_${PN} += "\
     genivi-dev-platform-hmi \
     qml-example \
     gdp-hmi-mouse-test \
+    gdp-new-hmi \
```

Comment:
I've asked Genivi for a name and have not been given anything different, this can be changed obviously at a later date